codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/LenghTraitCommon.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.rust.codegen.server.smithy.generators
+
+import software.amazon.smithy.model.traits.LengthTrait
+
+fun LengthTrait.rustCondition(): String {
```

Comment:
I'd suggest taking the variable name as argument, otherwise `length` is a bit of magical string here.

Reply:
Good shout!
Done in https://github.com/awslabs/smithy-rs/pull/2028/commits/92ceba8a0af16c83d5caf8ea1b076880d43f93f1.